### DISCUSSION TITLE
Alphafold integrated to google colab and others.

### DISCUSSION THREAD

**Discussion by superantichrist** [KosinskiLab/AlphaPulldown#168] [Ideas]
First, great work! I'm interested in aging pathway (currently specifically FGF2 related signaling) and with your program, I could find structure of multimer not yet determined in pdb.

Anyway, I only have 2080 ti local gpu card, I want to use this beatiful batch job in google colab. It will be wonderful to use this in colab.

And you also searched reactome pathway. Is there any plan to publish your other pathway result besides pathway in paper?

And one more question. Activated multimer protein transition it's structure and also interact with 3rd, 4rd protein. What could be good procedure for that interaction? list all 3~4 protein sequence to pulldown list? Is there any way of use advantage of already determined multimer to that?
ex) my procedure for FGFR_FGF multimer with GRB2_SOS1 interaction prediction.
run_multimer_jobs.py --mode=pulldown \
    --num_cycle=3 \
    --num_predictions_per_model=1 \
    --output_path=/media/numu/Z/bio/mutimer \
    --data_dir=/m2/alphafold_data \
    --protein_lists=FGFR_FGF.txt,GRB2_SOS1.txt \
    --monomer_objects_dir=/m2/AlphaPulldown/output \
    
[GRB2_SOS1.txt](https://github.com/KosinskiLab/AlphaPulldown/files/12438313/GRB2_SOS1.txt)
[FGFR_FGF.txt](https://github.com/KosinskiLab/AlphaPulldown/files/12438315/FGFR_FGF.txt)

Thanks

**Comment by jkosinski** [comment]
hey, @superantichrist, sorry for not responding, which of the above issues you have still unresolved?
